models/ovm1/optimism/ovm1_optimism_schema.yml

```diff
@@ -0,0 +1,206 @@
+version: 2
+
+models:
+  - name: ovm1_optimism_contracts
+    meta:
+      blockchain: optimism
+      project: ovm1
+      contibutors: MSilb7, chuxinh
+    config:
+      tags: ["optimism", "ovm1", "contracts"]
+    description: "Contracts deployed on OVM1."
+    columns:
+      - &name: contract_address
```

Comment:
- &name: contract_name

should be replaced by 

- &contract_name
        name: contract_name

in all & instances in this file